examples/structured_2d_dgsem/elixir_euler_warm_bubble.jl

```diff
@@ -0,0 +1,123 @@
+using OrdinaryDiffEq
+using Trixi
+
+# Physical constants
+g::Float64 = 9.81          # gravity of earth
+p_0::Float64 = 100_000.0   # reference pressure
+c_p::Float64 = 1004.0      # heat capacity for constant pressure (dry air)
+c_v::Float64 = 717.0       # heat capacity for constant volume (dry air)
+R = c_p - c_v              # gas constant (dry air)
+gamma = c_p / c_v          # heat capacity ratio (dry air)
+
+# Warm bubble test from
+# Wicker, L. J., and Skamarock, W. C., 1998: A time-splitting scheme
+# for the elastic equations incorporating second-order Runge–Kutta
+# time differencing. Mon. Wea. Rev., 126, 1992–1999.
+function initial_condition_warm_bubble(x, t, equations::CompressibleEulerEquations2D)
+    # center of perturbation
+    xc = 10000.0
+    zc = 2000.0
+    # radius of perturbation
+    rc = 2000.0
+    # distance of current x to center of perturbation
+    r = sqrt((x[1] - xc)^2 + (x[2] - zc)^2)
+
+    # perturbation in potential temperature
+    θ_ref = 300.0
+    Δθ = 0.0
```

Comment:
Summoning @sloede to let him praise ASCII for consistency (although I don't have objections agains unicode)